src/pg/sql/41_observatory_augmentation.sql

```diff
@@ -322,12 +446,13 @@ $$ LANGUAGE plpgsql;
 CREATE OR REPLACE FUNCTION cdb_observatory._OBS_GetPoints(
   geom geometry,
   geom_table_name text,
-  data_table_info cdb_observatory.OBS_ColumnData[]
+  data_table_info json[]
 )
-RETURNS NUMERIC[]
+RETURNS json[]
 AS $$
 DECLARE
   result NUMERIC[];
+  json_result json[];
   query  text;
   i int;
   geoid text;
```

Comment:
Yes.  There is a TODO elsewhere in the repo when we make a query with `geoid` as a hard-coded column name.  Internally, it would be best if we used the term `geometry_id`, I think, when naming our variables, although there's nothing technically wrong with using the variable name `geoid`.